benchmark/src/main/content/bin/kcb.sh

```diff
@@ -29,7 +29,7 @@ DIRNAME=$(dirname "$RESOLVED_NAME")
 
 # Default values
 JAVA_OPTS="-server"
-JAVA_OPTS="${JAVA_OPTS} -Xmx1G -XX:+HeapDumpOnOutOfMemoryError"
+JAVA_OPTS="${JAVA_OPTS} -Xmx4G -XX:+HeapDumpOnOutOfMemoryError"
```

Comment:
I saw the gatling java process up in the 3.5GB range when ramping up to 100k users